ocm_python_wrapper/cluster.py

```diff
@@ -249,33 +250,69 @@ def addon_info(self):
             self.addon_name
         ).to_dict()
 
-    def validate_addon_parameters(self, parameters):
+    def drop_unsupported_parameters(self, parameters=[]):
```

Comment:
please rename; the function filters required parameters for specific conditions

Reply:
done